draft-ietf-quic-transport.md

```diff
@@ -2096,7 +2096,7 @@ more likely to indicate an intentional migration rather than an attack.
 ## Loss Detection and Congestion Control {#migration-cc}
 
 The capacity available on the new path might not be the same as the old path.
-Packets sent on the old path SHOULD NOT contribute to congestion control or RTT
+Packets sent on the old path MUST NOT contribute to congestion control or RTT
 estimation for the new path.
 
```

Comment:
What about the initial RTT estimate on the new path?

Reply:
That's a fair point.  In practice, it's more likely the new path is similar to the old path RTT than it is to a default like 1 second, given these are the same two endpoints.

But I'm not sure that optimization is worth considering.  If you're validating the new path before migrating, which is probably the typical case, you expect to get an RTT sample in 1 round trip anyway.

Reply:
Yes, good question. So I think it is correct that it is important to make sure that when you migrate you need to use a new cc context and not just keep using the old one. But I guess this was a SHOULD because using the old estimate as the new initial could make sense e.g. if you have very small RTTs or only a few ms. On the other hand, as Ian said, you should get a new sample with the validation anyway and therefore it should not have an impact. Do we maybe need more text to actually explain that?

Reply:
I'd tend to agree with Ian that it's probably not necessary to add text covering that -- you're doing the safest thing if you treat them as completely independent, and you're likely to have new data from the new path just from validation or within a few RTTs.

(That said, I'd be quite interested in the results of experiments in this area, and there's no enforcement here that prevents such experimentation.)